packages/plugin-interactive-tools/sources/commands/search.tsx

```diff
@@ -0,0 +1,245 @@
+            </Box>
+            <Box marginLeft={1}>
+             Press <Color bold cyanBright>{`<space>`}</Color> to select a package.
+            </Box>
+            <Box marginLeft={1}>
+              Press <Color bold cyanBright>{`<space>`}</Color> again to change the target.
+            </Box>
+          </Box>
+          <Box flexDirection="column">
+            <Box marginLeft={1}>
+              Press <Color bold cyanBright>{`<enter>`}</Color> to install the selected packages.
+            </Box>
+            <Box marginLeft={1}>
+             Press <Color bold cyanBright>{`<ctrl+c>`}</Color> to abort.
+            </Box>
+          </Box>
+        </Box>
+      );
+    };
+
+    const SearchColumnNames = () => {
+      return <>
+        <Box width={17}><Color bold underline gray>Owner</Color></Box>
+        <Box width={17}><Color bold underline gray>Version</Color></Box>
+      </>;
+    };
+
+    const SelectedColumnNames = () => {
+      return <Box width={17}><Color bold underline gray>Target</Color></Box>;
+    };
+
+    const HitEntry = ({hit, active}: {hit: AlgoliaPackage, active: boolean}) => {
+      const [action, setAction] = useMinistore<string | null>(hit.name, null);
+
+      useSpace({
+        active,
+        handler: () => {
+          if (!action) {
+            setAction(targets[0]);
+            return;
+          }
+
+          const nextIndex = targets.indexOf(action) + 1;
+
+          if (nextIndex === targets.length) {
+            setAction(null);
+          } else {
+            setAction(targets[nextIndex]);
+          }
+        },
+      });
+
+      const ident = structUtils.parseIdent(hit.name);
+      const prettyIdent = structUtils.prettyIdent(configuration, ident);
+
+      return <Box>
+        <Box width={45} textWrap="wrap">
+          <Text bold>
+            {prettyIdent}
+          </Text>
+        </Box>
+        <Box width={16} textWrap="truncate" marginLeft={1}>
+          <Text bold>
+            {hit.owner.name}
+          </Text>
+        </Box>
+        <Box width={16} textWrap="truncate" marginLeft={1}>
+          {hit.version}
+        </Box>
+      </Box>;
+    };
+
+    const SelectedEntry = ({name, active}: {name: string, active: boolean}) => {
+      const [action] = useMinistore<string | null>(name, null);
+
+      const ident = structUtils.parseIdent(name);
+
+      return <Box>
+        <Box width={47}>
+          <Text bold>
+            {` - `}{structUtils.prettyIdent(configuration, ident)}
+          </Text>
+        </Box>
+        {targets.map(
+          target =>
+            <Box key={target} width={14} marginLeft={1}>
+              {action === target ? <Color green> ◉ </Color> : <Color yellow> ◯ </Color>}
+              <Text bold>{target}</Text>
+            </Box>
+        )}
+      </Box>;
+    };
+
+    const PoweredByAlgolia = () => {
+      return <Box marginTop={1}>
+        <Text>Powered by <Color blue>Algolia</Color>.</Text>
```

Comment:
The `blue` looks a little low-contrast to me. I think it looks fine just in one color.

Reply:
Turned it all white.